pkg/habitatctl/commands/community.go

```diff
@@ -69,12 +69,13 @@ var communityCreateCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 
 		address := cmd.Flags().Lookup("address")
-		if address == nil {
-			fmt.Println("address flag needs to be set")
+		name := cmd.Flags().Lookup("name")
+		if address == nil || name == nil {
+			fmt.Println("address and name flags need to be set")
 			return
 		}
 
-		SendRequest(ctl.CommandCommunityCreate, []string{address.Value.String()})
+		SendRequest(ctl.CommandCommunityCreate, []string{name.Value.String(), address.Value.String()})
```

Comment:
Note for the future, we definitely should turn these into protobufs in the future